client/my-sites/domains/domain-management/components/domain/non-transferrable-domain-notice.tsx

```diff
@@ -0,0 +1,17 @@
+import { useTranslate } from 'i18n-calypso';
+import Notice from 'calypso/components/notice';
+
+const NonTransferrableDomainNotice = ( { domainName }: { domainName: string } ): JSX.Element => {
+	const translate = useTranslate();
+	const text = translate(
+		"{{strong}}%(domain)s{{/strong}} is already past its redemption period so it's not possible to transfer it.",
```

Comment:
Shouldn't this be something like "...is in redemption so it's not possible to transfer it."?

Reply:
Indeed! Fixing this